docs/new-curriculum/js-core-1/week-1/notes.md

```diff
@@ -0,0 +1,349 @@
+
+:::note
+A **data type** is a grouping of data with some particular properties.
+:::
+
+In JavaScript, we have several different data types, like **numbers** and **strings**.
+
+For example, `10` is part of the number data type.  
+`3.14` is also part of the number data type, as we include both integers non-integers in this data type.
+
+We also have the string data type in JavaScript. A string is a sequence of characters. In JavaScript a string is denoted using opening quotation marks and closing quotation marks.
+
+### Creating expressions
+
+Let's consider the numbers 10 and 32. Given these 2 numbers, we might want to ask several questions about these numbers:
+
+a) What is the **sum** of 10 and 32?  
+b) What is the **product** of 10 and 32?  
+c) What is the **remainder** when 32 is divided by 10?
+
+From question a), instead of saying "the sum of 10 and 32" we can rewrite this description using numbers and a symbol. So “the sum of 10 and 32” can be written as `10 + 32` in symbolic form.
+
+```raw title="written form"
+the sum of 10 and 32
+```
+
+```raw title="symbolic form"
+10 + 32
+```
+
+In symbolic form, `+` is an operator: an **operator** is a symbol that is used to represent some kind of operation. In this example, `+` is used to represent the add operation “add the 2 numbers together”.  
+The actual combination of symbols `10 + 32` we say is an **expression**.
+
+:::info
+An **expression** is a value or any valid combination of values and symbols that results in a single value.  
+We say that expressions **evaluate** to a single value.  
+So we say that `10 + 32` **evaluates** to the value `42`.
+:::
+
+### Evaluating expressions
+
+So we can look at an expression like `36 * 45` and ask: what does this expression evaluate to? If we understand what the `*` operator represents ( in this case multiplication ) and if we understand some basic rules of arithmetic we can evaluate this expression.  
+However, we can use computers to evaluate expressions. We can use NodeJS, a JavaScript runtime, an application that runs JavaScript programs.
+
+Node can be used in a number of different ways; however, one particular mode is the REPL. The Node REPL enables us to input JavaScript expressions which are then evaluated by NodeJS and printed to the terminal. The REPL will print out the result of this evaluation.
+
+:::note
+
+ℹ️ Definition: REPL is a special type of program that stands for
+
+**Read**  
+**Evaluate**  
+**Print**  
+**Loop**
+
+:::
+
+We can type each of the following expressions into the REPL and then press enter to check the result.
+
+```bash
+10 + 32
+```
+
+```bash
+32 / 10
+```
+
+```bash
+3 ** 4
+```
+
+### Saving expressions
+
+Often in programming we may want to reuse a particular expression in our program. In this case, we need a way to refer back to a particular expression. Let’s consider the following string:
+
+```js
+"Hello there";
+```
```

Comment:
This is the first usage of a string expression.

I suggested above that we perhaps should cover number expressions first, then string expressions. But I now wonder if perhaps it would be useful to go through number expressions **and** variables, then come back to strings?

I think I would prefer to order it so that the concepts of expressions and variables come before the specifics of numbers and strings?